hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -187,6 +187,14 @@ public HoodieTimeline filterPendingReplaceTimeline() {
         s -> s.getAction().equals(HoodieTimeline.REPLACE_COMMIT_ACTION) && !s.isCompleted()), details);
   }
 
+  @Override
+  public HoodieTimeline filterPendingCommitTimeline() {
```

Comment:
Instead of adding new API. can you use 
timeline.getCommitsTimeline().filterInflightsAndRequested()

We do not want to add new APIs in HoodieTimeline unless it is needed.

Reply:
Thanks for you advices，had addressed it @bvaradar